foundry/foundry.py

```diff
@@ -725,7 +731,7 @@ def get_keys(self, type=None, as_object=False):
             return key_list
 
 
-    def _load_data(self, file=None, source_id=None, globus=True):
+    def _load_data(self, file=None, source_id=None, as_hdf5=False):
```

Comment:
did you remove `globus=True` because it was unused? It appears that is a mistake from a past merge -- please add it back in and we will create a separate issue to use `globus` 🙂

Reply:
Yes, I did remove it for that reason. However, I don't really see how it could be used, as all the data should already be loaded.

Reply:
Done